### DISCUSSION TITLE
running on Windows and run Elasticsearch from terminal (not docker)

### DISCUSSION THREAD

**Discussion by Aibat1997** [Jeroen-G/Explorer#220] [General]
Hi)
I have laravel 10 and run elasticsearch from terminal on Windows 10 (I downloaded elasticsearch from officail site https://www.elastic.co/downloads/elasticsearch). The problem is that when I search for the first two or three letters the result is not returned, but the result is returned when I search for the whole word.

**Comment by Jeroen-G** [comment]
Try your query in Kibana to find out where the issue is

**Reply by Aibat1997** [reply]
![image](https://github.com/Jeroen-G/Explorer/assets/34008555/f7b92d2b-70fd-4f3c-8c58-451b43fe180e)
Now I have this error, although elasticsearch and kibana running correctly. I did everything that was written in your instructions (implement Explorer, add driver elastic and add my model to indexes)

**Reply by Aibat1997** [reply]
![image](https://github.com/Jeroen-G/Explorer/assets/34008555/c76c3b5d-aa10-4f78-b345-36ccbbf8a5f4)

**Reply by Jeroen-G** [reply]
It says the order index does not exist. Can be ES or the search command that is wrong.

**Reply by Aibat1997** [reply]
But all comands is correct

**Comment by Aibat1997** [comment]
I fixed it by disabling SCOUT_QUEUE=false, but for some reason it doesn’t search for one letter at a time, to display search results you need to enter more than 4-5 letters
